### PR TITLE
Cosmos: Changelog updates for February 2023 release

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Updated `azure-cosmos` to version `4.41.0`.
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:12]: I am questioning whether we should release encryption or just wait for the major version release. @aayush3011  thoughts?

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 2]
```diff
@@ -5,15 +5,20 @@
...
-* Added improvement in `RntbdClientChannelHealthChecker` for detecting transit timeout. - See [PR 33464](https://github.com/Azure/azure-sdk-for-java/pull/33464)
 
 #### Other Changes
-* Give a meaningful name to the GlobalEndpointManager worker thread - See [PR 33507](https://github.com/Azure/azure-sdk-for-java/pull/33507)
+* Give a meaningful name to the GlobalEndpointManager worker thread. - See [PR 33507](https://github.com/Azure/azure-sdk-for-java/pull/33507)
+* Adding activity id in header of gateway address refresh call. - See [PR 33074](https://github.com/Azure/azure-sdk-for-java/pull/33074)
+* Direct mode - `RNTBD` connection health check improvements in `RntbdClientChannelHealthChecker` to allow recovering quicker when existing connections get broken (without TCP close or reset, just timeouts because packets get dropped). - See [PR 33464](https://github.com/Azure/azure-sdk-for-java/pull/33464) and - See [PR 33566](https://github.com/Azure/azure-sdk-for-java/pull/33566)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:21]: thanks :)

File: sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md (hunk 1 above)
Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-encryption/CHANGELOG.md:12]: We should have one without the breaking change
And then a new major version against the same azure-cosmos version also including the possibly breaking change

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Pull request merged**